types/codemirror/addon/hint/show-hint.d.ts

```diff
@@ -61,8 +61,8 @@ declare module "codemirror" {
         closeOnUnfocus?: boolean;
         completeOnSingleClick?: boolean;
         container?: HTMLElement | null;
-        customKeys?: {[key: string]: (editor: Editor, handle: Handle) => void} | null;
-        extraKeys?: {[key: string]: (editor: Editor, handle: Handle) => void} | null;
+        customKeys?: {[key: string]: (editor: Editor, handle: Handle) => void | string} | null;
+        extraKeys?: {[key: string]: (editor: Editor, handle: Handle) => void | string} | null;
```

Comment:
```suggestion
        extraKeys?: {[key: string]: ((editor: Editor, handle: Handle) => void) | string} | null;
```

I find that this type is inferred as a function returning 'string' or 'void'.
`((editor: Editor, handle: Handle) => void) | string` is suitable, I think. 🙇 

https://www.typescriptlang.org/play/#code/C4TwDgpgBAKlC8UDeAoK6oG0DWEQC4oBnYAJwEsA7AcwF1CAKASgQD4oA3Ae3IBMoAPsTJVqKAL6ColAK4AbOQG4UAYy6USUAGZcuhOIlQYoAIwCGpQgHISFGlYA0ExUA